doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1724,107 +1685,133 @@ String notations
 Tactic Notations
 -----------------
 
-Tactic notations allow to customize the syntax of tactics. They have the following syntax:
+Tactic notations allow customizing the syntax of tactics.
+
+.. todo move to the Ltac chapter
+
+.. cmd:: Tactic Notation {? ( at level @num ) } {+ @ltac_production_item } := @ltac_expr
+
+   .. insertprodn ltac_production_item ltac_production_item
+
+   .. prodn::
+      ltac_production_item ::= @string
+      | @ident {? ( @ident {? , @string } ) }
+
+   Defines a *tactic notation*, which extends the parsing and pretty-printing of tactics.
+
+   This command supports the :attr:`local` attribute, which limits the notation to the
+   current module.
 
-.. productionlist:: coq
-   tacn                 : Tactic Notation [`tactic_level`] [`prod_item` … `prod_item`] := `tactic`.
-   prod_item            : `string` | `tactic_argument_type`(`ident`)
-   tactic_level         : (at level `num`)
-   tactic_argument_type : ident | simple_intropattern | reference
-                        : hyp | hyp_list | ne_hyp_list
-                        : constr | uconstr | constr_list | ne_constr_list
-                        : integer | integer_list | ne_integer_list
-                        : int_or_var | int_or_var_list | ne_int_or_var_list
-                        : tactic | tactic0 | tactic1 | tactic2 | tactic3
-                        : tactic4 | tactic5
+   .. todo PR any words of wisdom on when to use tactic notation vs ltac?
+      can you run into problems if you shadow another tactic or tactic notation?
+      If so, how avoid ambiguity?
```

Comment:
Tactic notations are more powerful because they modify the parser instead of just defining a new object. But no, I don't have any wisdom to share. I think this kind of discussion should be delayed until we do take care of improving the Ltac chapter, and moving this section there. Then, we can ask the specialists.